nixos/tests/restic.nix

```diff
@@ -129,8 +129,8 @@ import ./make-test-python.nix (
 
           # test that custompackage runs both `restic backup` and `restic check` with reasonable commandlines
           "systemctl start restic-backups-custompackage.service",
-          "grep 'backup .* /opt' /tmp/fake-restic.log",
-          "grep 'check .* --some-check-option' /tmp/fake-restic.log",
+          "grep 'backup.* /opt' /tmp/fake-restic.log",
+          "grep 'check.* --some-check-option' /tmp/fake-restic.log",
```

Comment:
why was this changed?

Reply:
Thanks for your feedback!

The old search string ` .* ` had spaces before and after an optional command line argument. However, with having spaces on both sides the shortest accepted string would be `  ` (two spaces) or, in other words, at least one argument was required.
Prior to this PR's changes, most restic commands had the `--cache-dir=%C/restic-backups-${name}` command line argument, which would satisfy the `grep` command in question. After moving the cache location to the environment variable, present now to all restic calls of this module and not only most of them, the default `restic backup` call now has no further command line arguments and also `restic check` has lost one.